src/message/messageActionSheet.js

```diff
@@ -141,6 +141,22 @@ const allButtonsRaw = {
   cancel,
 };
 
+const errorMessages = [
+  'Failed to add reaction',
+  'Failed to reply',
+  'Failed to copy message to clipboard',
+  'Failed to share message',
+  'Failed to edit message',
+  'Failed to delete message',
+  'Failed to star message',
+  'Failed to unstar message',
+  'Failed to show reactions',
+  'Failed to unmute topic',
+  'Failed to mute topic',
+  'Failed to mute stream',
+  'Failed to unmute stream',
+];
```

Comment:
There's nothing keeping this lined up with the actions themselves. If your earlier `delete topic` PR were applied, for example, this would be suddenly and quietly broken.

It would be safer to associate actions with failure messages in the same way they're associated with titles, as I suggested earlier.

Reply:
Can you please give me an example of how you expect the alert titles to be associated, because I am unable to understand what we want here. Thanks. 

I Hardcoded the error titles because you mentioned that this is expected in a comment. But I guess you meant something else.